python/lsst/afw/display/interface.py

```diff
@@ -644,6 +644,26 @@ def erase(self):
         """
         self._impl._erase()
 
+    def centroids(self, catalog, symbol=None, *args, **kwargs):
```

Comment:
Why can't you do `symbol="o"`? It's not modifiable, like a list, right?

Reply:
Because I was dumb. Good call!